### DISCUSSION TITLE
Set a time period other than 24 hours as the default

### DISCUSSION THREAD

**Discussion by thegodfatherrelish** [alexjustesen/speedtest-tracker#1860] [Ideas]
Hello, at the moment, when you open the panel, the 24 hours are automatically displayed as the default value for the display of past tests. I would like to be able to set it in the environment variables so that the page is automatically called up with i. e. the weekly or monthly view, is that somehow possible? Thank you and have a nice weekend.

**Comment by svenvg93** [comment]
😉 custom ranges planned, attaching to https://github.com/alexjustesen/speedtest-tracker/issues/738

**Reply by thegodfatherrelish** [reply]
Oh crap, then I was looking for the wrong thing. Thanks, I'll be looking forward to the new feature soon!
